docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+services:
+  db:
+    image: postgres:16-alpine3.18
+    environment:
+      POSTGRES_PASSWORD: calypso
+
+  web:
+    build: .
+    environment:
+      JDBC_DATABASE_URL: "jdbc:postgresql://db:5432/postgres"
+      JDBC_DATABASE_USERNAME: postgres
+      JDBC_DATABASE_PASSWORD: calypso
+      APPLICATION_URL: http://localhost.datasektionen.se:3000
```

Comment:
According to the README this is the url used for the backend, so it might not really matter whether this is `localhost` or `localhost.datasektionen.se`? But should work either way so better safe than sorry

Reply:
It is used when generating the RSS feeds and for the callback url to login, so it will be fine to use localhost when [#35](https://github.com/datasektionen/login/pull/35) is merged